mast/prelude/ast_printer.mt

```diff
@@ -1148,7 +1108,8 @@ bind printerActions :Map[Str, DeepFrozen] := [
         def pattern := self.getPattern()
         if (priorities["braceExpr"] < priority) {
             if (pattern.getNodeName() == "FinalPattern") {
-                if (pattern.getGuard() == null && isIdentifier(pattern.getNoun().getName())) {
+                if (!maybe(pattern.getGuard(), fn _ { true }) &&
```

Comment:
I'm not really in love with any of this logic, but I guess that this is the price of wanting to get `@foo` instead of `@{foo}`, which is what I think it's trying to do.

Reply:
yeah, it's a little twisted

Reply:
oh... `!null` is no good in monte, is it...